package.json

```diff
@@ -10,6 +10,9 @@
         "publish": "npm install && npm run build && npm publish --access public"
     },
     "dependencies": {
+        "@material-ui/core": "^4.11.0",
+        "dotenv": "^8.2.0",
```

Comment:
I think these two packages(material-ui, dotenv) are not used to this component.
How about removing it?

Reply:
I agree with you.

Reply:
Please remove these packages.